app/javascript/components/import/Import.js

```diff
@@ -74,10 +86,25 @@ class Import extends React.Component {
     this.setState({ importDuplicates: value });
   }
 
+  stopImport = async () => {
+    this.setState({ isPaused: true });
+    let confirmText = `This will stop the  creation of new records from the list below. Records that were imported prior to clicking “Stop Import” will not be deleted from the system.`;
+    if (await confirmDialog(confirmText, { title: 'Stop Import Process' })) {
+      this.setState({ isPaused: false });
+      location.href = '/';
+    } else {
+      this.setState({ isPaused: false });
+      if (this.state.acceptedAllStarted) {
+        this.submit(this.state.phased[this.state.progress], this.state.progress, true);
```

Comment:
This is creating duplicates of the record that was just finished. You need to increment `this.state.progress`. `this.state.progress` appears to me to be "the last index that was successfully finished", and this call is saying, "I want to submit this record next".